linkingFuncs.py

```diff
@@ -33,11 +33,43 @@ def get_profile(username):
     cur = cnx.cursor(buffered=True)
     id = get_id(username, cur)
     #get transactions
-    cur.execute("SELECT * FROM Transactions WHERE UserId='%d'" % (id))
+    cur.execute("SELECT * FROM Transactions WHERE UserId='%d' ORDER BY ticker" % (id))
     fetch = cur.fetchall()
     if (fetch == []):
         return { 'Action': False }
-    print(fetch)
+    ret = []
+    oldTik = ""
+    volume = 0
+    totalSpent = 0
+    for result in fetch:
```

Comment:
Can you possibly add a comment indicating the contents of result to provide some context on what is being accessed when indexing into result[i]?

Reply:
Something that could also be nice to make it more readable would be to define constants for the indexes that describe what they're indexing. I.e if result[1] returns the volume, then you could define VOLUME = 1 at the top and then access result[VOLUME].

Reply:
Consider defining variables as Jonathan has described earlier.

Reply:
Sounds good I will make sure to indicate what the variables mean!